ca/config.go

```diff
@@ -274,6 +274,9 @@ func LoadSecurityConfig(ctx context.Context, rootCA RootCA, krw *KeyReadWriter)
 
 	// Check to see if this certificate was signed by our CA, and isn't expired
 	if _, err := X509Cert.Verify(opts); err != nil {
+		fmtString := "Jan 02 15:04:05 2006 MST"
+		log.G(ctx).WithError(err).Debugf("invalid certificate issued by %s (not before %s, not after %s)",
+			X509Cert.Issuer.CommonName, X509Cert.NotBefore.Format(fmtString), X509Cert.NotAfter.Format(fmtString))
```

Comment:
I think it would be great to compare `NotBefore` / `NotAfter` to the current wall clock time, and include the current time in the error message if it looks like a factor. It may not be obvious when a clock is out of sync.